tails_files/install.sh

```diff
@@ -20,15 +40,44 @@ apt-get update
 apt-get install -y build-essential
 gcc -o $SCRIPT_BIN securedrop_init.c
 
-# copy init scripts
-cp securedrop_init.py $SCRIPT_PY
-cp .xsessionrc $INSTALL_DIR
-
+# copy icon, launchers and scripts
 cp securedrop_icon.png $INSTALL_DIR
+cp document.desktop $INSTALL_DIR
+cp source.desktop $INSTALL_DIR
+cp .xsessionrc $INSTALL_DIR
+cp securedrop_init.py $SCRIPT_PY
 
-# prepare torrc_additions
-cp torrc_additions $ADDITIONS
-gedit $ADDITIONS > /dev/null 2>&1
+if $ADMIN; then
+  DOCUMENT=`cat $ANSIBLE/app-document-aths | cut -d ' ' -f 2`
+  SOURCE=`cat $ANSIBLE/app-source-ths`
+  APPSSH=`cat $ANSIBLE/app-ssh-aths | cut -d ' ' -f 2`
+  MONSSH=`cat $ANSIBLE/mon-ssh-aths | cut -d ' ' -f 2`
+  cat $ANSIBLE/app-ssh-aths $ANSIBLE/mon-ssh-aths $ANSIBLE/app-document-aths >> $ADDITIONS
+  if [[ -d "$HOMEDIR/.ssh" && ! -f "$HOMEDIR/.ssh/config" ]]; then
+    # create SSH host aliases and install them
+    SSHUSER=$(zenity --entry --title="Admin SSH user" --window-icon=$INSTALL_DIR/securedrop_icon.png --text="Enter your username on the App and Monitor server:")
```

Comment:
Another note on this line: we can actually extract the SSH username from `ssh_users` in `prod-specific.yml`. However, the admin should know the username they used on the servers at this point in the installation and I don't see getting rid of this prompt as a requirement to merge this PR. Noting it here because it might be a nice future improvement.